### ISSUE TITLE
add of two ore more review tags does not work correctly

### ISSUE THREAD

**Issue by biancode** [AgileReview-Project/AgileReview-Legacy-Plugin#24]
new issue started from [AgileReview-EclipsePlugin](https://github.com/AgileReview-Project/AgileReview-EclipsePlugin/issues/5)

![grafik](https://user-images.githubusercontent.com/4369867/36838652-0ac5aab6-1d40-11e8-87e3-bc2e8464171b.png)

**Comment by biancode** [comment]
a) add review line 85 - works well
b) add review line 79 - smashing until next review and starts from line 60 instead of 79 and span to line 85

That behavior is mostly in context with comments in file like  `/*`

**Comment by biancode** [comment]
somtimes also just one review tag goes on and on if there are two or more tags

![grafik](https://user-images.githubusercontent.com/4369867/36839057-2eb35cc4-1d41-11e8-8172-09e5e4025f41.png)

**Comment by maybeec** [comment]
Can you provide your `language support` configuration for the specific file?
Especially, the first example may result in an issue if for the kind of files you are discussing here has `/*` and `*/` configured as start and end tags for comments. As your file type already accepts `/*` as a one line comment, the comment is never closed for the AgileReview parser.

The second example is a little more strange for me. The only situation when this occurs might be the situation if you set the cursor at the end of the commented line and creating new lines from there. The yellow annotation will be automatically expanded due to invisible markers in the editor, which are refreshed on save. Is this also occuring on non-dirty (saved) edtiors?

**Comment by biancode** [comment]
a) yes `/*` is an inline comment without the `*/` and in a new line the `*/` is code and not a comment anymore

![grafik](https://user-images.githubusercontent.com/4369867/39295139-efa85e70-493d-11e8-94a2-155dae95168a.png)


![grafik](https://user-images.githubusercontent.com/4369867/39295242-29fc6d78-493e-11e8-8c80-0b8179302791.png)

**Comment by maybeec** [comment]
This basically is an issue of missing support for single line comments.
You most probably added a new file ending of your format in the yellow marked line to make it work somehow.

![unbenannt](https://user-images.githubusercontent.com/1427255/39297847-e16f2d1e-4944-11e8-8d4b-1eb66c8b4425.PNG)

But as the dialog states, `/*` and `*/` will be interpreted as multiline comment syntax. 
There is currently no support for single line comments in AgileReview.
This is also related to #19.

**Comment by biancode** [comment]
is that allowed?

![grafik](https://user-images.githubusercontent.com/4369867/39298848-9922c25c-4947-11e8-9cd0-98321b6aebf7.png)

**Comment by maybeec** [comment]
No, but implementing a similar solution might be an option. I'll check that
